src/content/protocols/sky.md

```diff
@@ -0,0 +1,397 @@
+---
+protocol: "Sky"
+website: "https://sky.money/"
+x: "https://x.com/SkyEcosystem"
+github: ["https://github.com/makerdao"]
+defillama_slug: ["sky-lending"]
+chain: "Ethereum"
+stage: 0
+reasons: []
+risks: ["L", "H", "H", "H", "L"]
+author: ["mmilien_"]
+submission_date: "2025-03-11"
+publish_date: "2025-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Sky is a stablecoin protocol allowing users to mint its USDS stablecoin through _Collateralized Debt Positions_ with a variety of crypto collateral assets. Sky is built on the Maker protocol, by the MakerDAO, to replace the DAI stablecoin and allow for the 1:1 conversion between DAI and USDS. USDS can further be staked for the yield-bearing sUSDS token. Various levels of control over the Sky protocol are exercised through an onchain governance system.
+
+# Ratings
+
+## Chain
+
+This review focuses on the Ethereum deployment of Sky, currently the only chain on which Sky protocol is deployed. Ethereum achieves a Low centralization risk score."
+
+> Chain score: Low
+
+## Upgradeability
+
+`USDS` and `sUSDS`, Sky's stablecoin and its corresponding yield-bearing version, are upgradeable contracts through Sky governance proposals. Updating those contracts could change the entire logic of those tokens and may incur a loss of funds for users.
+
+Many parameters in the Sky Protocol can be changed through governance proposals and may also incur loss of funds, loss of unclaimed yield, as well as materially impact the protocol's performance. Example actions are forced liquidations, which would result in loss of user funds, creating unbacked debt, which could endanger the protocol's stability, or pausing the contracts, which could trap user funds for an undetermined amount of time.
+
+> Upgradeability score: High
+
+## Autonomy
+
+Sky has a centralized dependency on Circle and its `USDC` stablecoin token. This is because users can mint `USDS` from `USDC` at a fixed 1:1 rate. This means that `USDS` is directly pegged to `USDC` (instead of `USD`), which is a centralized stablecoin. This conversion may be stopped or paused in an emergency Sky Governance proposal. There is a debt ceiling limiting how much `USDS` can be backed by `USDC`. Nonetheless, at the time of writing, this debt ceiling is high enough that it does not prevent more than 50% of the collateral in Sky from being backed by `USDC`. The ceiling is explained further in the [dependencies](#dependencies) section.
+
+The Sky protocol relies on the provider Chronicle for price feeds of collateral assets. The Chronicol protocol uses a system of validators hosted by various third parties such as Bitcoin Suisse, ETHGlobal, Gitcoin, and Etherscan. Validators push price updates on-chain. We found that any changes to the validator set are subject to a 7 days exit window. We therefore assessed that Chronicle is a Stage 1 dependency and explained it in detail in a [dedicated report](./chronicle.md).
```

Comment:
The Sky protocol further relies on Chronicle's oracle price feeds to assess the solvency of debt positions.

Reply:
"Chronicle thus achieves a _Medium_ centralization risk score as discussed in a separate report [here](./chronicle.md)."